fix(windows): Typo in autoplay browser arg (#1287)
tauri-apps/wry: .changes/windows-autoplay.md

---
wry: patch
---

> Fixed an issue causing autoplay on windows to require user gestures again.

Reviewer: ```suggestion
Fixed a regression causing autoplay on windows to require user gestures.
```
